src/content/energy_consumption/index.md

```diff
@@ -0,0 +1,78 @@
+
+## The merge
+
+There is now a functional PoS blockchain called the [Beacon Chain]("https://ethereum.org/en/eth2/beacon-chain/") that has been running since December 2020 that is demonstrating the viability of the PoS protocol. Now that the ["London Upgrade"](https://ethereum.org/en/history/#london) has been successfully implemented, the switch from PoW to Pos (known as "the merge") is the next major update coming to Ethereum with a target date of late 2021 - early 2022. The merge to PoS will also bring associated efficiency gains through "sharding" the blockchain into sub-chains and accelerating the adoption of "roll-ups" (collections of transactions settled cheaply off-chain then pushed to the Ethereum blockchain as a single batch transaction).
+
+## PoS energy expenditure
+
+As well as building confidence in the PoS mechanism, the Beacon Chain also enables estimates of Ethereum's post-merge energy usage. A recent [blog post on this site](https://blog.ethereum.org/2021/05/18/country-power-no-more/) suggested that the merge to PoS could result in a 99.95% reduction in total energy use, with PoS being ~2000x more efficient than PoS. The energy-expenditure of Ethereum will be roughly equal to the cost of running a home computer for each node on the network, estimated to be about 5,250,000kWh per year for a network of 10,000 nodes.
+
+![image](energy_use_per_transaction.png)
+
+We can use this data to compare Eth2 to a global service like Visa. 100,000 Visa transactions uses 149kWh of energy<sup>[^2]</sup>. In Eth2, that same number of transactions would cost 17.4kWh of energy or ~11% of the total energy<sup>[^3]</sup>. That's without considering the many optimisations being worked on in parallel to Eth2, like [rollups](/glossary/#rollups). It could be as little as 0.1666666667kWh of energy for 100,000 transactions.
+
+It's estimated that Eth2 will allow the network to process between 25,000 and [100,000 transactions per second](https://twitter.com/VitalikButerin/status/1312905884549300224?ref_src=twsrc%5Etfw%7Ctwcamp%5Etweetembed%7Ctwterm%5E1312905886327664640%7Ctwgr%5E%7Ctwcon%5Es2_&ref_url=https%3A%2F%2Fwww.coinspeaker.com%2Fvitalik-buterin-ethereum-layer-2%2F), with [100,000 as the theoretical maximum right now](https://ethereum-magicians.org/t/a-rollup-centric-ethereum-roadmap/4698).
+At the bare minimum, Eth2 will allow 64 times the amount of transactions as today which sits at around 15 transactions. That's the amount of shard chains (extra data and capacity) being introduced ([More on shard chains](/eth2/shard-chains/)). That means we can conservatively estimate how long it will take to process 100,000 transactions so we can compare it to the Visa example above.
```

Comment:
```suggestion
At a minimum, Eth2 will allow 64 times the amount of transactions as today which sits at around 15 transactions. That's the number of [shard chains](/eth2/shard-chains/) (extra data and capacity) being introduced. That means we can conservatively estimate how long it will take to process 100,000 transactions so we can compare it to the Visa example above.
```

Reply:
Also: we should probably be explaining that shared chains allow this growth (not anything else in the eth2 updates) especially because [we're moving away from the ETH2 branding](https://notes.ethereum.org/@timbeiko/great-renaming)